apps/site/next.config.mjs

```diff
@@ -96,6 +97,23 @@ const nextConfig = {
       'shiki',
     ],
   },
+  webpack: (config, options) => {
+    if (process.env.NEXT_PUBLIC_ANALYZE) {
```

Comment:
Why is this a NEXT_PUBLIC prefixed variable? `NEXT_PUBLIC` are meant for env vars you want available during the runtime. This seems to be only used during build time.